feature_server/config/scripts/grownade.py

```diff
@@ -97,6 +91,23 @@ def model_grenades(connection, expression = None):
                 loaded.append(filename)
             else:
                 failed.append(filename)
+    return models, loaded, failed
+
+@name('model')
+@admin
+def model_grenades(connection, expression = None):
+    protocol = connection.protocol
+    if connection not in protocol.players:
+        raise ValueError()
+    player = connection
+
+    result = None
+    if expression:
+        try:
+            models, loaded, failed = load_models(expression)
+        except ModelLoadFailure as err:
+            result = str(err)
+        else:
```

Comment:
Yep definitely - I think I've fixed it, but I guess the only way to check for logic problems would be to thoroughly test the script